index.test.ts

```diff
@@ -138,40 +138,48 @@ function makeTest(args: string) {
 }
 
 test("simple", async () => {
-  const {stdout, stderr, exitCode} = await execa(script, [
+  const subprocess = spawn(process.execPath, [
```

Comment:
```js
spawn(script, ...)
```

fails on Windows.

Reply:
Okay. We have Windows on CI, which probably should have caught this.